### DISCUSSION TITLE
Ability to have no search engine option?

### DISCUSSION THREAD

**Discussion by michaerlrob92** [TarekRaafat/autoComplete.js#313] [Q&A] [answered]
I've got autoComplete.js fetching data from an api endpoint that searches an index with the passed query. 

Is there anyway to turn off the search engine in autoComplete.js so that it returns all of the data from the endpoint?

Thanks

**Comment by folknor** [comment, marked as answer]
This is the same as the last submitted question.

https://github.com/TarekRaafat/autoComplete.js/discussions/310

**Reply by michaerlrob92** [reply]
Oh thank you, I didn't see that for some reason. Looked at all the other questions 😦
